apps/chat/src/components/Header/BackToChatMobile.tsx

```diff
@@ -0,0 +1,33 @@
+import { IconArrowLeft } from '@tabler/icons-react';
+
+import { useTranslation } from 'next-i18next';
+import { useRouter } from 'next/router';
+
+import { Translation } from '@/src/types/translation';
+
+import Tooltip from '../Common/Tooltip';
+
+export const BackToChatMobile = () => {
+  const { t } = useTranslation(Translation.Header);
+
+  const router = useRouter();
+
+  return (
+    <Tooltip isTriggerClickable tooltip={t('Back to Chat')}>
```

Comment:
do we need tooltip here?

Reply:
@Alexander-Kezik , I suppose so, because we have collapsed view